lambda/main.go

```diff
@@ -146,3 +157,41 @@ func invalidateCloudFrontCache(session *session.Session) error {
 
 	return err
 }
+
+func getStatus(isFailure bool) string {
```

Comment:
This function can be removed. Not being used anymore

Reply:
sure @namandf